LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2011,2012,2013,2014,2015,2016,1017 Genome Research Ltd.
+Copyright (c) 2011,2012,2013,2014,2015,2016,2017 Genome Research Ltd.
```

Comment:
This licence is repeated in the header of a few of the files, but with the wrong version declared.  We might want to remove those headers elsewhere.